src/02_evaluate_better_models.R

```diff
@@ -27,17 +27,17 @@ for(i_renglon in 1:n_renglones) {
                                              parametrosModelo = tabla_coeficientes_auxiliar)
   
   readr::write_lines(jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), 
```

Comment:
En el primer ejemplo de[ esta](https://style.tidyverse.org/syntax.html#long-lines) guía sugieren el siguinete cambio:
```suggestion
  readr::write_lines(
      jsonlite::toJSON(listaParametrosModeloNormalizacion, pretty = T), 
```